roles/matrix-riot-web/defaults/main.yml

```diff
@@ -32,6 +32,25 @@ matrix_riot_web_welcome_user_id: "@riot-bot:matrix.org"
 # Branding of riot web
 matrix_riot_web_brand: "Riot"
 
+# URL to Logo on welcome page
+matrix_riot_web_welcome_logo: "welcome/images/logo.svg"
+
+# URL of link on welcome image
+matrix_riot_web_welcome_logo_link: "https://riot.im"
+
+matrix_riot_web_welcome_headline: "_t('Welcome to Riot.im')"
+matrix_riot_web_welcome_text: "_t('Decentralised, encrypted chat &amp; collaboration powered by [matrix]')"
+
+# Links, shown in footer of welcome page:
+# [{"text": "Link text", "url": "https://link.target"}, {"text": "Other link"}]
+matrix_riot_web_welcome_footerlinks: false
+
+# URL to image, shown during Login
+matrix_riot_web_welcome_authlogo: "{{ matrix_riot_web_welcome_logo }}"
+
+# URL to Wallpaper, shown in background of welcome page
+matrix_riot_web_welcome_background: false
```

Comment:
Likewise, I'd like to confirm that `false` is a good value for what would normally be a string.

What if we use an empty string or null (`~`)?